R/data_import.R

```diff
@@ -88,6 +88,17 @@ import_data <- function(dataset = NULL,
 
   # Step two: move raw data file to correct location
   if (is.null(path)) path <- file.choose()
+  # Make sure names are consistent
+  if (!grepl(paste0(dataset), path)) {
+    stop("Please make sure that raw data file and dataset have the same name for consistency.")
```

Comment:
Relates to our conversation on Friday about whether it is better to have everything internally consistent or bring things as close as possible to e.g. importing the raw data as downloaded from the website.